app/helpers/application_helper.rb

```diff
@@ -21,6 +21,10 @@ def display_date(date)
     date.strftime(STANDARD_DATE_DISPLAY_FORMAT)
   end
 
+  def display_singular_or_plural_str(pluralized_string, count)
+    count == 1 ? pluralized_string.singularize : pluralized_string
+  end
```

Comment:
Can we use [Rails's pluralization](https://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize) instead?

Reply:
Wow, I swear I searched for this...